sources/packages/backend/workflow/src/workflow-definitions/parttime-configure-disbursement.bpmn

```diff
@@ -108,7 +108,9 @@
+          <zeebe:output source="={&#10;  CSLPAmount: round half up(finalFederalAwardNetCSLPAmount / 2, 0),&#10;  CSGPAmount: round half up(finalFederalAwardNetCSGPAmount / 2, 0),&#10;  CSGDAmount: round half up(finalFederalAwardNetCSGDAmount / 2, 0),&#10;  CSPTAmount: round half up(finalFederalAwardNetCSPTAmount / 2, 0),&#10;  BCAGAmount: round half up(finalProvincialAwardNetBCAGAmount / 2, 0),&#10;  SBSDAmount: round half up(finalProvincialAwardNetSBSDAmount / 2, 0)&#10;}" target="firstHalfDisbursementValues" />
+          <zeebe:output source="={&#10;  CSLPAmount: ceiling(floor(finalFederalAwardNetCSLPAmount) - firstHalfDisbursementValues.CSLPAmount),&#10;  CSGPAmount: ceiling(floor(finalFederalAwardNetCSGPAmount) - firstHalfDisbursementValues.CSGPAmount),&#10;  CSGDAmount: ceiling(floor(finalFederalAwardNetCSGDAmount) - firstHalfDisbursementValues.CSGDAmount),&#10;  CSPTAmount: ceiling(floor(finalFederalAwardNetCSPTAmount) - firstHalfDisbursementValues.CSPTAmount),&#10;  BCAGAmount: ceiling(floor(finalProvincialAwardNetBCAGAmount) - firstHalfDisbursementValues.BCAGAmount),&#10;  SBSDAmount: ceiling(floor(finalProvincialAwardNetSBSDAmount) - firstHalfDisbursementValues.SBSDAmount)&#10;}" target="secondHalfdisbursementValues" />
```

Comment:
Was there a reason to provide the second parameter as zero? It seems to be the default behavior already unless I am missing something.
```
round half up(finalFederalAwardNetCSLPAmount / 2, 0)
```